### PR TITLE
Jixie Bid adapter: support more userids

### PR DESCRIPTION
**Pull request by jxdeveloper1**
## Type of change
- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
added code to sending tdid and uid2 and any publisher id to our backend

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by patmmccann** [comment]
Please add docs

**Comment by jxdeveloper1** [comment]
Hi folks, I might need a bit of guidance here.
 
To my PR (initial) I received a comment (earlier commit) from Patrick McCann saying "Please add docs". 

Since this pull request is about jixie changing to send userids to our backend, I looked at other bidders who are sending userids to their backends - to see what kind of comments/docs they have on this topic. I was unable to find.

So here, I only added a rather lame comment in the md file (With this latest commit (updating the PR) I have actually changed the implementation too). Please let me know what kind of thing I should write about. THANKYOU!

**Comment by patmmccann** [comment]
You should submit an accompanying pr for this change to this file https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/jixie.md in which you disclose your support for some userids. See https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/rtbhouse.md for an example disclosure.

**Comment by jxdeveloper1** [comment]
corresponding PR in the doc repo:
https://github.com/prebid/prebid.github.io/pull/3734
updated the jixie.md file

File: modules/jixieBidAdapter.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { BANNER, VIDEO } from '../src/mediaTypes.js';
 import { ajax } from '../src/ajax.js';
 import { getRefererInfo } from '../src/refererDetection.js';
 import { Renderer } from '../src/Renderer.js';
+import {createEidsArray} from './userId/eids.js';
```
Comment (patmmccann) [inline modules/jixieBidAdapter.js:9]: @dgirardi please note another location this is occurring

**Review by patmmccann — APPROVED** [review]

**Pull request merged**